src/deepsparse/evaluation/registry.py

```diff
@@ -11,12 +11,71 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+Implementation of a registry for evaluation functions
+"""
+from typing import Any, Callable, List, Optional, Union
 
 from sparsezoo.utils.registry import RegistryMixin
 
 
 __all__ = ["EvaluationRegistry"]
 
 
+def _merge_name_alias(
```

Comment:
thoughts on just adding alias to the mixin?

Reply:
Potentially beneficial. 
Maybe better idea would be to refactor `RegistryMixin` so it has the following UI:

```python
# register with default name
@Dataset.register()
class ImageNetDataset(Dataset):
    pass

# register with custom name
@Dataset.register(name="cifar-dataset")
class Cifar(Dataset):
    pass
    
# register with multiple names
@Dataset.register(name=["cifar-dataset", "some_alias"])
class Cifar(Dataset):
    pass
```
I find it more elegant and generic.

If you like it, confirm and I can implement it asap (I will need to tweak the `RegistryMixing` source code to small extent anyway)

Reply:
Seconded that having multiple aliases would be useful in the `RegistryMixin`

Reply:
Cool. Vox populi vox dei. I'll implement it :)